app/controllers/alchemy/admin/user_sessions_controller.rb

```diff
@@ -1,6 +1,11 @@
 module Alchemy
   module Admin
     class UserSessionsController < ::Devise::SessionsController
+      include Alchemy::ConfigurationMethods
+      include Alchemy::AbilityHelper
+      include Alchemy::ControllerActions
```

Comment:
The only module we need is the `Alchemy::ControllerActions` as this provides the `current_alchemy_user` method. All other modules should not be included.

Reply:
Actually this module is already included in `Alchemy::BaseController` and the `Devise::SessionsController` inherits from that. I wonder why the method is not available 🤔 It should be.